openedx/openedx-learning
# feat: add object tag view permissions [FC-0036]

## Description
Add view permission for ObjectTag

- The `oel_tagging.view_objecttag_objectid` rule is used in the rest API view to check if the user has permission on the object_id to see its tags

## Supporting Information
* https://github.com/openedx/modular-learning/issues/129

## Testing instructions
* Please ensure that the tests cover the expected behavior of the view
____
Private-ref: [FAL-3518](https://tasks.opencraft.com/browse/FAL-3518)
